.github/workflows/upstream-build.yml

```diff
@@ -43,7 +43,17 @@ jobs:
           west blobs fetch hal_silabs
 
       - name: Build Bluetooth sample for xG27
+        continue-on-error: true
         working-directory: zephyr-silabs
         shell: bash
         run: |
           west build -b xg27_dk2602a/efr32bg27c140f768im40 ../zephyr/samples/bluetooth/peripheral_hr
+
+      - name: Build hello world for siwx917_rb4338a
+        working-directory: zephyr-silabs
+        shell: bash
+        run: |
+          if [ "${{ runner.os }}" = "Windows" ]; then
+            EXTRA_TWISTER_FLAGS="--short-build-path -O/tmp/twister-out"
+          fi
```

Comment:
Any reason to not apply these flags on Linux/Macos?

Reply:
> Any reason to not apply these flags on Linux/Macos?

I assume this is related to Windows-specific filesystem properties. The convention is simply copied from upstream: https://github.com/zephyrproject-rtos/zephyr/blob/c44abdf5b7932b080f31e90cebbec07a8e092068/.github/workflows/hello_world_multiplatform.yaml#L68